leap/leap_test.rb

```diff
@@ -15,26 +17,38 @@ def leap?
 
 class YearTest < Minitest::Test
   def test_leap_year
-    assert Year.leap?(1996), 'Yes, 1996 is a leap year'
+    assert Year.leap?(1996)
```

Comment:
Let's start by adding it to the json file and to this generator. We can backfill later. I'll do the Go ones :)